bocadillo/applications.py

```diff
@@ -199,6 +204,12 @@ def __init__(
         self.add_asgi_middleware(
             TrustedHostMiddleware, allowed_hosts=allowed_hosts
         )
+        if enable_sessions:
+            secret_key = os.environ.get("SECRET_KEY", "")
+            if secret_key:
+                self.add_asgi_middleware(
+                    SessionMiddleware, secret_key=secret_key
+                )
```

Comment:
Shouldn't we fail loudly here if the `SECRET_KEY`? The user knowingly opted in to use sessions, so I believe we should refuse to start if the `SECRET_KEY` is not set. Otherwise this could lead to using empty secret keys in production without knowing.

So, something like:
```python
secret_key = os.environ.get("SECRET_KEY", "")
if not secret_key:
    raise Exception(
        "The 'SECREY_KEY' environment variable must be set and non-empty to use sessions."
    )
```